source/features/clean-repo-filelist-actions.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'dom-chef';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	// Only select desktop version of the button
+	observe('.d-flex > [data-hotkey="t"]:not(.rgh-clean-actions)', {
+		add(searchButton) {
+			searchButton.classList.add('tooltipped', 'tooltipped-ne', 'rgh-clean-actions');
+			searchButton.setAttribute('aria-label', 'Go to file');
+
+			searchButton.firstChild!.replaceWith(<SearchIcon/>);
+
+			// Exclude logged out, mobile or file pages
+			const addButtonText = searchButton.nextElementSibling?.querySelector('.d-md-flex');
+			if (addButtonText) {
+				addButtonText.parentElement!.classList.add('tooltipped', 'tooltipped-ne');
+				addButtonText.parentElement!.setAttribute('aria-label', 'Add file');
+
+				addButtonText.classList.replace('d-md-flex', 'd-md-block');
+				addButtonText.firstChild!.replaceWith(<PlusIcon/>);
+			}
+		}
+	});
+
+	observe('get-repo summary:not(.rgh-clean-actions)', {
+		add(button) {
+			button.classList.add('tooltipped', 'tooltipped-ne', 'rgh-clean-actions');
```

Comment:
Could you check if _click outside to close_ works?. That's the reason I moved that classes from `<summary>` to `<details>` [here](https://github.com/sindresorhus/refined-github/commit/b501d71a83857a7068d8e8c1ca5ef21bd7a93233#diff-5ba5120afe524f2d5eed502b044996a1db29ba24221b41edee322940746bf553R17).